File: src/1078.occurrences-after-bigram.1156/solution.js
/**
 * @param {string} text
 * @param {string} first
 * @param {string} second
 * @return {string[]}
 */
var findOcurrences = function (text, first, second) {
    const list = text.split(' ');
    const result = [];
    for (let i = 0; i < list.length - 2; i++) {
        if (list[i] === first && list[i + 1] === second) {
            result.push(list[i + 2]);
        }
    }
    return result;
};

File: src/1078.occurrences-after-bigram.1156/solution.java
class Solution {
    public String[] findOcurrences(String text, String first, String second) {
        String[] texts = text.split(" ");
        List<String> result = new ArrayList<>();
        for(int i=0;i<texts.length-2;i++){
            if(texts[i].equals(first) && texts[i+1].equals(second)){
                result.add(texts[i+2]);
            }
        }

        return result.toArray(new String[0]);
    }
}